source/nodejs/adaptivecards-designer/src/tool-box.ts

```diff
@@ -94,6 +94,7 @@ export class Toolbox {
 
         let headerTitleElement = document.createElement("span");
         headerTitleElement.className = "acd-toolbox-header-title";
+        headerTitleElement.id = this.title.replace(/ /g, "-");
```

Comment:
I would try to avoid this hack way. It may introduce localization and other issue.

also why do you set an id for it?

Reply:
Just updated :)